h/util/metrics.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import unicode_literals
+import newrelic.agent
+
+
+def send_metric(name, metric):
+    """ Send a metric to New Relic.
+
+    :arg name: The key name. Only the first 255 characters are retained
+    :type name: str
+    :arg metric: The string value to add to the current transaction
+    Only the first 255 characters are retained.
+    :type metric: str, int, float, bool
+    """
+    newrelic.agent.add_custom_parameter(name, metric)
+
+
+def record_search_query_param_usage(params, separate_replies):
+    """ Send metric for each parameter if it matches list of desired keys.
+
+    :arg params: the search parameters that will be popped by each of the filters
+    :type params: webob.multidict.MultiDict
+    :arg separate_replies: Records usage of _separate_replies parameter in the search
+    :type separate_replies: bool
+    """
+    # Record usage of search params and associate them with a transaction.
```

Comment:
```suggestion
```

I think we should delete this comment as it seems superfluous, just repeating what the docstring already said. Also I'm not sure what the "transaction" that it's referring to is, presumably some New Relic concept (?), but we probably don't need to mention that detail in our code.